vspec2x.py

```diff
@@ -199,11 +202,11 @@ def processDataTypeTree(parser: argparse.ArgumentParser, args, include_dirs,
     Helper function to process command line arguments and invoke logic for processing data
     type information provided in vspec format
     """
-    # check that both required arguments are available to process the data
-    # type tree
-    if args.types_output_file is None or args.vspec_types_file is None:
+    if args.vspec_types_file is None:
```

Comment:
This shall never evaluate to true as we check this before we call, right? But I do not mind having the check here as well, but does it make sense to continue if this evaluates to true? I.e. shall we add a return after the error below. We also need to rephrase the error message below and remove "and the output file"

Reply:
You are right. The check is redundant. I'll remove it.